includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php

```diff
@@ -24,13 +24,44 @@ public static function init() {
 			return;
 		}
 
+		// Add suggestions to the product tabs.
+		add_action( 'woocommerce_product_data_tabs', array( __CLASS__, 'product_data_tabs' ) );
+		add_action( 'woocommerce_product_data_panels', array( __CLASS__, 'product_data_panels' ) );
+
 		// Register ajax api handlers.
 		add_action( 'wp_ajax_woocommerce_add_dismissed_marketplace_suggestion', array( __CLASS__, 'post_add_dismissed_suggestion_handler' ) );
 
 		// Register hooks for rendering suggestions container markup.
 		add_action( 'wc_marketplace_suggestions_products_empty_state', array( __CLASS__, 'render_products_list_empty_state' ) );
 	}
 
+	/**
+	 * Product data tabs filter
+	 *
+	 * Adds a new Extensions tab to the product data meta box.
+	 *
+	 * @param array $tabs Existing tabs.
+	 *
+	 * @return array
+	 */
+	public static function product_data_tabs( $tabs ) {
+		$tabs['marketplace-suggestions'] = array(
+			'label'    => _x( 'Get more options', 'Marketplace suggestions', 'woocommerce' ),
```

Comment:
See more options? Not keen on this copy :)

Reply:
Probably best to follow that up on P2.

https://woomarketplace.wordpress.com/2019/02/15/extension-discovery-all-screens-i2/

https://woomarketplace.wordpress.com/2019/02/15/extension-discovery-all-screens-i2/#comment-2131